java/algorithm/pom.xml

```diff
@@ -48,5 +48,49 @@
     </dependency>
   </dependencies>
 
-  <build></build>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <formats>
+            <format>
```

Comment:
assuming we keep license formatting for pom.xml only for this module, we can still merge with the parent pom.xml by adding `combine.children="append"` attribute to the `<pom>` element (and remove the need to specify the list of files to include as those are automatically deduced from the section)

Reply:
I will check this out. :+1:

Reply:
Since we differed to do this in another PR for all pom's. I will skip this for the moment.